src/behaviours/quitting-and-restarting-the-app/opening-application-window-using-tray.test.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import type { ApplicationBuilder } from "../../renderer/components/test-utils/get-application-builder";
+import { getApplicationBuilder } from "../../renderer/components/test-utils/get-application-builder";
+import { lensWindowInjectionToken } from "../../main/start-main-application/lens-window/application-window/lens-window-injection-token";
+import applicationWindowInjectable from "../../main/start-main-application/lens-window/application-window/application-window.injectable";
+import createElectronWindowForInjectable from "../../main/start-main-application/lens-window/application-window/create-electron-window-for.injectable";
+import type { AsyncFnMock } from "@async-fn/jest";
+import asyncFn from "@async-fn/jest";
```

Comment:
_nit:_ 

```suggestion
import asyncFn, { type AsyncFnMock } from "@async-fn/jest";
```

Reply:
We're using auto import + eslint which makes two separate lines. So if we want to switch to using suggested, let's update eslint to do the work for us. :)